packages/grafana-ui/src/components/Drawer/Drawer.tsx

```diff
@@ -154,16 +155,16 @@ export function Drawer({
               />
             </div>
             {typeof title === 'string' ? (
-              <div className={styles.titleWrapper}>
-                <Text element="h3" {...titleProps}>
+              <Stack direction="column">
+                <Text element="h3" truncate {...titleProps}>
                   {title}
                 </Text>
                 {subtitle && (
-                  <div className={styles.subtitle} data-testid={selectors.components.Drawer.General.subtitle}>
+                  <Text element="p" color="secondary" truncate>
```

Comment:
given `subtitle` can be anything, i don't think we can safely wrap it in `Text`. for `title`, we're explicitly checking it's a string